doc/release/release_dev.rst

```diff
@@ -1,16 +1,44 @@
-*****************************************
-Version 2.0 release notes and API changes
-*****************************************
+Announcement: networkx 2.0
+==========================
 
-This page includes more detailed release information and API changes from
-NetworkX 1.10 to NetworkX 2.0.
+We're happy to announce the release of networkx 2.0!
 
-There is a `migration guide for people moving from 1.X to 2.0 <http://networkx.readthedocs.org/en/latest/release/migration_guide_from_1.x_to_2.0.html>`_.
+networkx is a Python package for the creation, manipulation, and study of the
+structure, dynamics, and functions of complex networks.
 
-Please send comments and questions to the networkx-discuss `mailing list <http://groups.google.com/group/networkx-discuss>`_.
+For more information, please visit our website
 
-API changes
+http://networkx.github.io/
+
+and our gallery of examples
+
+http://networkx.readthedocs.io/en/latest/auto_examples/index.html
+
+Please send comments and questions to the networkx-discuss `mailing list
+<http://groups.google.com/group/networkx-discuss>`_.
+
+Highlights
+----------
+
+This release is the result of X of work, with over X pull requests by
+X contributors. Highlights include:
+
+- big items ...
+
+There is a `migration guide for people moving from 1.X to 2.0
+<http://networkx.readthedocs.org/en/latest/release/migration_guide_from_1.x_to_2.0.html>`_.
```

Comment:
We need to update this doc. This page was written when `G.nodes()`, `G.edges()`, `G.degree()`, .... were changed to return iterators and now these functions return views.